homeassistant/components/switcher_kis/coordinator.py

```diff
@@ -12,7 +12,7 @@
 from homeassistant.helpers import device_registry as dr, update_coordinator
 from homeassistant.helpers.dispatcher import async_dispatcher_send
 
-from .const import DOMAIN, MAX_UPDATE_INTERVAL_SEC, SIGNAL_DEVICE_ADD
+from .const import CONF_TOKEN, DOMAIN, MAX_UPDATE_INTERVAL_SEC, SIGNAL_DEVICE_ADD
```

Comment:
`from homeassistant.const import CONF_TOKEN`

Reply:
It is done.